crates/solver/src/settlement_submission/submitter.rs

```diff
@@ -367,10 +385,10 @@ impl<'a> Submitter<'a> {
 
         let mut access_list: Option<AccessList> = None;
 
-        loop {
-            // Try to find submitted transaction from previous submission loop (with the same address and nonce)
-            let pending_gas_price = transactions.last().cloned().map(|(_, gas_price)| gas_price);
+        // Try to find submitted transaction from previous submission loop (with the same address and nonce)
```

Comment:
Ubernit: 
Maybe you want to remove the word "loop": it made a lot of sense before, but now you are referring to  transactions from previous submissions/
But you can also keep it in there. It would still make sense.

Reply:
Since this comment is so close to the `loop` keyword but doesn't really refer to that specific loop it might indeed be a bit confusing. Changed it to `attempt`.